RAM_KUMAR/assignment/Calculator/calculatorunittest.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from calculator import Calculator
+
+class TestCalculator(unittest.TestCase):
+
+  def setUp(self):
+    self.calculator = Calculator()
+
+  def test_add(self):
+    self.assertEqual(self.calculator.add(4,7), 11)
+  def test_subtract(self):
+    self.assertEqual(self.calculator.subtract(10,5), 5)
+  def test_multiply(self):
+    self.assertEqual(self.calculator.multiply(3,7), 21)
+  def test_divide(self):
+    self.assertEqual(self.calculator.divide(10,2), 5)
+
+if __name__ == "__main__":
+  unittest.main()
```

Comment:
It's good that you tried writing unit tests using unittest framework. But you need to use pytest here. So please change all the unit test as per pytest